schema/crdb/dbinit.sql

```diff
@@ -444,7 +448,9 @@ CREATE TABLE IF NOT EXISTS omicron.public.physical_disk (
     sled_id UUID NOT NULL,
 
     disk_policy omicron.public.physical_disk_policy NOT NULL,
-    disk_state omicron.public.physical_disk_state NOT NULL
+    disk_state omicron.public.physical_disk_state NOT NULL,
+
+    CONSTRAINT physical_disk_variant_u2 CHECK (variant = 'u2')
```

Comment:
I can imagine this looking kinda weird in isolation - it might be worth a comment linking to the issue?